src/platform/forms-system/src/js/review/ArrayField.jsx

```diff
@@ -331,7 +336,7 @@ class ArrayField extends React.Component {
                 <button
                   type="button"
                   disabled={addAnotherDisabled}
-                  className="edit-btn primary-outline"
+                  className="add-btn primary-outline"
```

Comment:
Uh....so this seems right, but...what? Any idea why this was `edit-btn` before?

Reply:
I think it's a copy-paste problem that didn't get updated. It wasn't a problem before since it didn't get focused.